modules/dxp/apps/portal-workflow/portal-workflow-metrics-api/src/main/java/com/liferay/portal/workflow/metrics/search/index/name/WorkflowMetricsIndexNameBuilder.java

```diff
@@ -14,10 +14,12 @@
 
 package com.liferay.portal.workflow.metrics.search.index.name;
 
-import com.liferay.portal.search.index.IndexNameBuilder;
-
 /**
  * @author Rafael Praxedes
  */
-public interface WorkflowMetricsIndexNameBuilder extends IndexNameBuilder {
+public interface WorkflowMetricsIndexNameBuilder {
+
+	public String getIndexName(
+		long companyId, WorkflowMetricsIndexEntityName indexEntityName);
```

Comment:
@W2jx `WorkflowMetricsIndexEntityName indexEntityName`, please rename the parameter to "workflowMetricsIndexEntityName"